static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -166,6 +166,28 @@ const registerIframeListener = (iframeId: string) => {
     });
 };
 
+const addServerSideRenderingTestParameterToLink = (rawUrl: string): string => {
+    const serverSideRenderingField = 'ssr';
+    const randomNumber = Math.random();
+    if (
+        randomNumber === undefined ||
+        config.get('switches.contributionsServerSideRenderingSwitch', false)
```

Comment:
This will `return rawUrl` if your switch is on 🙃 So you may have to `!` it or move around your logic:

```js
if (
    typeof randomNumber === 'number' && 
    config.get('switches.contributionsServerSideRenderingSwitch', false)
) { 
    // do stuff here e.g. addParamValueToSearchParams and return url.toString
```

Reply:
👍